Core/Object Arts/Dolphin/Sockets/InternetAddressTest.cls

```diff
@@ -17,19 +17,17 @@ testAllForHost
 	"Microsoft should have more than one IP associated with microsoft.com"
 	addresses := InternetAddress allForHost: 'microsoft.com'.
 	self assert: addresses size > 1.
-	self assert: (addresses anySatisfy: [:each | each ipAddress beginsWith: #[191 239]]).
 
 	"OA central"
 	addresses := InternetAddress allForHost: 'object-arts.com'.
-	self assert: addresses size = 1.
-	self assert: (addresses first host endsWith: 'cable.virginm.net')!
+	self assert: addresses size = 1!
 
 testHost
 	| name ip |
 	self isOnline ifFalse: [^self].
 	name := InternetAddress host: 'uk2.net'.
 	ip := InternetAddress ipAddress: name ipAddress.
-	self assert: ip host = name host!
+	self assert: (ip host endsWith: name host)!
 
 testIsIPString
 	self deny: (InternetAddress isIPString: '').
```

Comment:
Replacing testLocalHost with something like the below will make it clear which of the assertions is failing:
```smalltalk
testLocalHost
	| local computerName |
	computerName := SessionManager current computerName asLowercase.
	local := InternetAddress localHost.
	self assert: local host asLowercase equals: computerName.
	self assert: ((InternetAddress ipAddress: local ipAddress) host subStrings: $.) first asLowercase
		equals: computerName
```